eng/vscode/README.md

```diff
@@ -128,7 +128,7 @@ The Azure MCP Server provides tools for interacting with **28 Azure service area
 - 🧮 **Azure Data Explorer** - Analytics queries and KQL
 - 🐘 **Azure Database for PostgreSQL** - PostgreSQL database management
 - 🛠️ **Azure Developer CLI (azd)** - Template and deployment management
-- 🧮 **Azure Foundry** - AI model management and deployment
+- 🧮 **Azure Foundry** - AI model management and deployment, knowledge index management
```

Comment:
Either change added comma into a semicolon  OR change the phrasing to "AI model management, AI model deployment, and knowledge index management"

Otherwise it's not correct grammar. I'd just change the phrasing so you can use all commas.